.circleci/config.yml

```
@@ -727,20 +727,24 @@ commands:
     steps:
       - run: python .circleci/scripts/wait_for_master.py <<parameters.scheme>>://<<parameters.host>>:<<parameters.port>>
 
-  upload-cloud-watch-to-s3:
+  locate-cloud-watch-logs:
     parameters:
       cluster-id:
         type: string
-      bucket:
+      region:
         type: string
-        default: det-ci-cloud-watch-logs
+        default: us-west-2
     steps:
       - run:
-          name: Upload cloud watch logs S3
+          name: Locate cloud watch logs
           when: always
           command: |
             LOG_GROUP=$(python .circleci/scripts/get_output_from_stack.py <<parameters.cluster-id>> LogGroup)
-            python .circleci/scripts/upload_cloud_watch_to_s3.py ${LOG_GROUP} <<parameters.bucket>> <<parameters.cluster-id>>
+            echo "Cluster logs can be found in Cloudwatch under the log group $LOG_GROUP"
+            echo "or the URL below (if the log group is in your default region):"
+            ENC_LOG_GROUP=$(echo $LOG_GROUP | sed "s/\//\$252F/g")
```

You can use single quotes instead of double and also a different character for the regex separator, saving some escaping. It's also preferable as a general rule to add quotes around the variable expansion like this (unless you want to split on whitespace), though it is unlikely to matter here.
```suggestion
            ENC_LOG_GROUP=$(echo "$LOG_GROUP" | sed 's|/|$252F|g')
```

---

🤷 I have no idea of the generally accepted style for these things